packages/gatsby-theme-docs/src/layouts/internals/layout-page-with-hero.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from '@emotion/styled';
+import { ThemeProvider } from 'emotion-theming';
+import { designSystem, Markdown } from '@commercetools-docs/ui-kit';
+
+// The container is used to render a background image with for the full page width.
+const Container = styled.div`
+  grid-area: page-header;
+  position: relative;
+  background-attachment: fixed;
+  background-size: cover;
+  background-color: ${(props) => props.theme.colors.light.primary};
+
+  /* stylelint-disable-next-line function-whitespace-after */
+  background-image: ${(props) => `url('${props.heroBackgroundURL}')`};
+
+  @media screen and (${designSystem.dimensions.viewports.tablet}) {
+    background-size: auto;
+  }
+  @media screen and (${designSystem.dimensions.viewports.largeDesktop}) {
+    background-size: cover;
+  }
+
+  /* License with attribution of the background by SVGBackgrounds.com */
```

Comment:
Wouldn't this comment better live in the folder where the images are?  It could be forgotten here once our own images are in.

Reply:
I'd rather keep it here for now, as we need to somehow mention the license and that is "visible" on production